models/public/midasnet/midasnet.md

```diff
@@ -25,15 +25,15 @@ See [here](https://github.com/intel-isl/MiDaS)
 | Metric            | Value         |
 |-------------------|---------------|
 | Type              | Monodepth     |
-| GFLOPs            | 207.4915      |
-| MParams           | 104.0814      |
+| GFLOPs            | 207.25144     |
+| MParams           | 104.081       |
```

Comment:
How could GFLOPS/MParams change if only the weights are different?

Reply:
I just checked it again with the current DL Workbench version and could those results.